Sources/PackageModel/ManifestSourceGeneration.swift

```diff
@@ -319,7 +319,7 @@ fileprivate extension SourceCodeFragment {
             }
             self.init(enum: "target", subnodes: params)
             
-        case .product(name: let name, package: let packageName, moduleAliases: let aliases, condition: let condition):
+        case .product(name: let name, package: let packageName, moduleAliases: _, condition: let condition):
```

Comment:
should this actually include this information in the SourceCodeFragments instead of ignoring it?

Reply:
It's tracked separately rdar://88722774

Reply:
ok